app.py

```diff
@@ -1,12 +1,36 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
-
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+#changed to the template for local host v2.0
+#unable to complete the sort by status function 
```

Comment:
Do you still want to do that?

To sort the list you will want to use `your_sorted_list = sorted(your_list, your_sorting_function)`, where your_sorting_function takes a single element and returns the value to sort on, e.g. a number or string.